pkg/datapath/linux/node.go

```diff
@@ -811,6 +805,14 @@ func (n *linuxNodeHandler) nodeUpdate(oldNode, newNode *nodeTypes.Node, firstAdd
 	return nil
 }
 
+func (n *linuxNodeHandler) getApproriateIfaceName() string {
```

Comment:
Typo in the function name? Also I would suggest to have a better less vague name for this function.

Reply:
Thanks, fixed.